README.md

```diff
@@ -71,7 +71,7 @@ The NATS server uses a [text based protocol](http://nats.io/documentation/intern
 Trying 107.170.221.32...
 Connected to demo.nats.io.
 Escape character is '^]'.
-INFO {"server_id":"mxyLRV3o96bPBziXld35wt","version":"1.0.6","go":"go1.8.4","host":"0.0.0.0","port":4222, ...}
+INFO {"server_id":"JUOufISZqT09TSan4iBHCJ","version":"1.0.6","git_commit":"","go":"go1.9.4","host":"0.0.0.0","port":4222,"auth_required":false,"ssl_required":false,"tls_required":false,"tls_verify":false,"max_payload":1048576}
```

Comment:
I wonder if we should deprecate ssl_required.

Reply:
You mean removed from INFO, but keep in structs that we have? If we remove from structs, people embedding the NATS Server will have a problem...

Reply:
humm... so we have a struct in client_test.go which has `TLSRequired` but uses `json:"ssl_required"`, so this one would be easy to fix. The other is in `client.go` and although this is a non exported struct we return it through `ClientAuthentication.GetOpts()`.. that is weird. That struct has only `SslRequired`.

Reply:
See if we can clean it up. I think those embedding NATS server will be ok on a recompile unless they explicitly set sslRequired.